### PR TITLE
feat: introducing telegram output

### PR DESCRIPTION
**Pull request by zufardhiyaulhaq**
**What type of PR is this?**
/kind feature

**Any specific area of the project related to this PR?**
/area outputs

**What this PR does / why we need it**:
- introducing telegram output
- some company might use telegram for alerting system

**Which issue(s) this PR fixes**:
N/A

**Special notes for your reviewer**:
tested on local
image
image
image

### CODE REVIEW

File: outputs/telegram.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+	)
+
+	return replacer.Replace(fmt.Sprintf("%v", text))
+}
+
+var (
+	telegramMarkdownV2Tmpl = `*\[Falco\] \[{{markdownV2EscapeText .Priority }}\] {{markdownV2EscapeText .Rule }}*
+
+• *Time*: {{markdownV2EscapeText .Time }}
+• *Source*: {{markdownV2EscapeText .Source }}
```
Comment (Issif) [inline outputs/telegram.go:29]: Can you also add the hostname field, please

**Comment by zufardhiyaulhaq** [comment]
done @Issif, please help check back
image

**Review by Issif — APPROVED** [review]

**Comment by zufardhiyaulhaq** [comment]
@Issif what is the next step? I don't see the workflow running.

can I start contributing to falcosidekick helm chart for this output?

**Comment by Issif** [comment]
> @Issif what is the next step? I don't see the workflow running.
> 
> can I start contributing to falcosidekick helm chart for this output?

I don't know why it didn't start. I closed/reopened the PR, all tests started.

**Comment by Issif** [comment]
> @Issif what is the next step? I don't see the workflow running.
> 
> can I start contributing to falcosidekick helm chart for this output?

For the helm chart update, we must wait for the release 2.28.0, we'll add the new settings at this moment.

**Pull request merged**